A patch to the tensorflow's patch file for eigen.

A recent upgrade of toolchains removes #include <sstream> from <complex>, which
breaks tensorflow build because Eigen/Core gets <sstream> from <complex>, see
https://crbug.com/1056770.

This patch file adds an explicit #include <sstream> to tensorflow's patch file
for eigen to fix this issue, that's to say, it patches a patch file.
It will first be used by ebuild system to patch
third_party/eigen3/gpu_packet_math.patch of tensorflow source code, then when
building tensorflow with bazel, gpu_packet_math.patch will be used to patch
eigen.
diff --git a/third_party/eigen3/gpu_packet_math.patch b/third_party/eigen3/gpu_packet_math.patch
index b8afb1b4..5d2bca2a 100644
--- a/third_party/eigen3/gpu_packet_math.patch
+++ b/third_party/eigen3/gpu_packet_math.patch
@@ -1,3 +1,13 @@
+--- a/Eigen/Core
++++ b/Eigen/Core
+@@ -41,6 +41,7 @@
+ #endif
+ 
+ #include <complex>
++#include <sstream>
+ 
+ // this include file manages BLAS and MKL related macros
+ // and inclusion of their respective header files
 --- a/Eigen/src/Core/arch/GPU/PacketMath.h
 +++ b/Eigen/src/Core/arch/GPU/PacketMath.h
 @@ -100,6 +100,7 @@
